mir-module/src/main/resources/META-INF/resources/js/ckeditor-config.js

```diff
@@ -0,0 +1,18 @@
+CKEDITOR.editorConfig = function( config ) {
```

Comment:
Codacy found an issue: ['CKEDITOR' is not defined.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=5834770)